### PR TITLE
Add repository field in package.json

### PR DESCRIPTION
**Pull request by MoOx**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "tape-jsx-equals",
   "version": "1.0.0",
   "description": "tape assertion to compare react components",
+  "repository": "https://github.com/atabel/tape-jsx-equals.git"
```
Comment (atabel) [inline package.json:5]: there is a missing comma

Comment (MoOx) [inline package.json:5]: oups.

Comment (MoOx) [inline package.json:5]: Maybe you can just close this PR and add it yourself? Faster than me cloning + force pushing a single commit (made from github in the first place :))

Comment (atabel) [inline package.json:5]: ok ;)

**Comment by atabel** [comment]
Fixed in cf219925e3f90a9fbbc96d7861f1d625b18d166d
Thanks ;)

**Pull request closed without merging**